tests/eas/acceptance_wake_migration.config

```diff
@@ -1,48 +0,0 @@
-{
-    "modules"  : [ "bl" ],
-    "exclude_modules"  : [ "hwmon" ],
-    "tools"    : [ "rt-app" ],
-    "ftrace" : {
-        "events" : [
-            "sched_overutilized",
-            "sched_energy_diff",
-            "sched_load_avg_task",
-            "sched_load_avg_cpu",
-            "sched_migrate_task",
-            "sched_switch",
-            "cpu_frequency",
-        ],
-    },
-    "wloads" : {
-        // Create one small task for each CPU
-        "wake_migration" : {
-            "type" : "rt-app",
-            "conf" : {
-                "class" : "profile",
-                "params" : {
-                    "wmig" : {
-                        "kind" : "Step",
-                        "params" : {
-                            "start_pct": 10, // STEP_LOW_DCYCLE
```

Comment:
Why we discarded the comments? Should these be translated into global variables for consistency?